deploy/helm/sumologic/values.yaml

```diff
@@ -649,7 +649,7 @@ sumologic:
           - port: metrics
         selector:
           matchLabels:
-            sumologic.com/app: otelcol-logs-collector
+            sumologic.com/app-name: otelcol-logs-collector
```

Comment:
Where does this label come from?

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/7c1008b7e8c68f3d27c1b7dd40cbe1b92adf8825/deploy/helm/sumologic/templates/logs/collector/otelcol/daemonset.yaml#L43

Reply:
But these selectors are for the Service, not the Daemonset.

Reply:
oh, I was trying figure out why e2e are failing :facepalm: